packages/fx-core/src/plugins/resource/bot/configs/teamsBotConfig.ts

```diff
@@ -61,6 +63,11 @@ export class TeamsBotConfig {
         if (capabilities?.includes(PluginActRoles.MessageExtension) && !this.actRoles.includes(PluginActRoles.MessageExtension)) {
             this.actRoles.push(PluginActRoles.MessageExtension);
         }
+
+        const resourceNameSuffixValue: ConfigValue = context.configOfOtherPlugins.get(PluginSolution.PLUGIN_NAME)?.get(PluginSolution.RESOURCE_NAME_SUFFIX);
+        if (resourceNameSuffixValue) {
```

Comment:
I would like your previous PR to put utils.genUUID together with setting this.resourceNameSuffix. What about your thought?

Reply:
Both are ok to me. Let me update it.